File: add/codesnippet/CSharp/m-system.net.networkinfo_3_1.cs
         public static void ShowActiveUdpListeners()
         {
             Console.WriteLine("Active UDP Listeners");
             IPGlobalProperties properties = IPGlobalProperties.GetIPGlobalProperties();
             IPEndPoint[] endPoints =  properties.GetActiveUdpListeners();
             foreach (IPEndPoint e in endPoints)
             {
                 Console.WriteLine(e.ToString());
             }
         }

File: add/codesnippet/CPP/m-system.net.networkinfo_3_1.cpp
void ShowActiveUdpListeners()
{
   Console::WriteLine( "Active UDP Listeners" );
   IPGlobalProperties ^ properties = IPGlobalProperties::GetIPGlobalProperties();
   array<IPEndPoint^>^endPoints = properties->GetActiveUdpListeners();
   System::Collections::IEnumerator^ myEnum8 = endPoints->GetEnumerator();
   while ( myEnum8->MoveNext() )
   {
      IPEndPoint^ e = safe_cast<IPEndPoint^>(myEnum8->Current);
      Console::WriteLine( e );
   }
}